src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -109,6 +110,35 @@ internal static bool IsPrimitiveType(this CorElementType et)
             // COR_ELEMENT_TYPE_I1,I2,I4,I8,U1,U2,U4,U8,R4,R8,I,U,CHAR,BOOLEAN
             => ((1 << (int)et) & 0b_0011_0000_0000_0011_1111_1111_1100) != 0;
 
+        private static ReadOnlySpan<short> PrimitiveWidenTable =>
```

Comment:
This is reflection helper. It should be in a reflection-related type like InvokeUtils (it is where it lives in naot). 

Also, this helper and PrimitiveWiden should be next to each other. They are closely related.